source/guide.html.haml

```diff
@@ -239,19 +239,36 @@ introduction: >
 
   - example do
     :plain
-      @mixin transform($property) {
-        -webkit-transform: $property;
-        -ms-transform: $property;
-        transform: $property;
+      @mixin theme($theme: DarkGray) {
+        background: $theme;
+        box-shadow: 0 0 1px rgba($theme, .25);
+        color: #fff;
+      }
+
+      .info {
+        @include theme;
+      }
+      .alert {
+        @include theme($theme: DarkRed);
+      }
+      .success {
+        @include theme($theme: DarkGreen);
       }
-      .box { @include transform(rotate(30deg)); }
       ===
-      =transform($property)
-        -webkit-transform: $property
-        -ms-transform: $property
-        transform: $property
-      .box
-        +transform(rotate(30deg))
+      @mixin theme($theme: DarkGray)
```

Comment:
are we discouraging the shorthand for the original syntax? (`=` & `+`)?

Reply:
ok no worries! I just know the previous example used them. but it's not necessary. :)